network-api/networkapi/wagtailpages/templates/wagtailpages/fragments/formassembly_body.html

```diff
@@ -0,0 +1,287 @@
+<!-- FORM: BODY SECTION -->
```

Comment:
FormAssembly snippet with minor modifications to pass value into form field's `value` attributes